src/main/java/org/candlepin/subscriptions/tally/collector/RHELProductUsageCollector.java

```diff
@@ -93,15 +94,26 @@ public Optional<HostTallyBucket> collectForHypervisor(
     if (appliedSockets == 0) {
       log.warn(
           "Hypervisor in account {} has no sockets and will"
-              + " not contribute to the totals. The tally for the RHEL product will not be accurate since"
-              + "all associated guests will not contribute to the tally.",
+              + " not contribute to the totals. The tally for the RHEL product will not be"
+              + " accurate since all associated guests will not contribute to the tally.",
           account);
     }
 
     prodCalc.addHypervisor(appliedCores, appliedSockets, 1);
-    return Optional.of(
-        createBucket(
-            prodCalc, false, appliedCores, appliedSockets, HardwareMeasurementType.VIRTUAL));
+    HardwareMeasurementType hardwareType = translateHardwareType(hypervisorFacts.getHardwareType());
+    return Optional.of(createBucket(prodCalc, true, appliedCores, appliedSockets, hardwareType));
+  }
```

Comment:
Actually, this is where I expected we would see HardwareMeasurementType.HYPERVISOR, something like

```
createBucket(prodCalc, false, appliedCores, appliedSockets, HardwareMeasurementType.HYPERVISOR));
```